Automattic/simplenote-macos
# Replace Fabric with sentry

Adds support for using Sentry via Tracks.

Most of the logic is over in Automattic/Automattic-Tracks-iOS#103. Once that's merged, I'll update this PR with a real version number, instead of a `pod` branch.

**To Test**

1. Put `CrashLogging.crash()` (you'll need to `import AutomatticTracks`) in `AboutViewController.swift` 
2. Create a release version of the app using Build > Archive. Export it to your desktop to run it.
3. Cause a crash by viewing the app's About Screen.
4. Re-run the app, and let it idle for a moment (this'll ensure the crash report goes to Sentry)
5. Note your shiny new crash entry in Sentry (it'll likely get attached to an existing sample crash from my testing. However yours should be the most recent event.
